package-res/resources/web/pentaho/type/config/AmdLoadedConfigurationService.js

```diff
@@ -15,14 +15,34 @@
  */
 define([
   "./ConfigurationService",
-  "pentaho/service!pentaho.type.ITypeConfiguration?meta"
+  "pentaho/service!pentaho.type.spec.ITypeConfiguration?meta"
 ], function(ConfigurationService, configurations) {
   "use strict";
 
   configurations.sort(function(a, b) {
     return a.moduleId.localeCompare(b.moduleId);
   });
 
+  /**
+   * @classDesc The `AmdLoadedConfigurationService` class is an implementation of
+   * the {@link pentaho.type.IConfigurationService} interface that
+   * contains, preloaded,
+   * all of the value type configurations that are registered as providing the
+   * service {@link pentaho.type.spec.ITypeConfiguration} (see {@link pentaho.service}).
+   *
```

Comment:
suggested:
The `AmdLoadedConfigurationService` class is an implementation of the {@link pentaho.type.IConfigurationService} interface. This class contains (preloaded) all of the value type configurations that are registered as providing the service {@link pentaho.type.spec.ITypeConfiguration} (see {@link pentaho.service}).